internal/kafka/command_topic_produce.go

```diff
@@ -33,36 +34,40 @@ func (c *command) newProduceCommand() *cobra.Command {
 		Args:              cobra.ExactArgs(1),
 		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
 		RunE:              c.produce,
-		Annotations:       map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogin},
 	}
 
 	cmd.Flags().String("key-schema", "", "The ID or filepath of the message key schema.")
 	cmd.Flags().String("schema", "", "The ID or filepath of the message value schema.")
 	pcmd.AddKeyFormatFlag(cmd)
 	pcmd.AddValueFormatFlag(cmd)
-	cmd.Flags().String("key-references", "", "The path to the message key schema references file.")
 	cmd.Flags().String("references", "", "The path to the message value schema references file.")
 	cmd.Flags().Bool("parse-key", false, "Parse key from the message.")
 	cmd.Flags().String("delimiter", ":", "The delimiter separating each key and value.")
 	cmd.Flags().StringSlice("config", nil, `A comma-separated list of configuration overrides ("key=value") for the producer client.`)
 	pcmd.AddProducerConfigFileFlag(cmd)
 	cmd.Flags().String("schema-registry-endpoint", "", "Endpoint for Schema Registry cluster.")
+	pcmd.AddOutputFlag(cmd) // Deprecated
+	cobra.CheckErr(cmd.Flags().MarkHidden("output"))
+
+	// cloud-only flag
+	cmd.Flags().String("kafka-bootstrap", "", "Bootstrap URL for Kafka cluster.")
```

Comment:
Should the description mention that this is only for Cloud Kafka clusters?

Reply:
Also, it looks like we're going to have both `--bootstrap` and `--kafka-bootstrap`. Can we mark `--kafka-bootstrap` as hidden and deprecate it? (... and open a ticket to remove it in V4)

Reply:
agree with updating the description. those 2 bootstraps are totally different, why are we removing `kafka-bootstrap`?

Reply:
Even though they're totally different, a user might be confused about what the difference is. Instead we should just use `--bootstrap` for everything so the user doesn't have to think about which one to use

Reply:
hmm `bootstrap` is part of standard on-prem authentication flags. i hesitate on mix-using them

Reply:
What's wrong with using the same flag for both cloud and on-prem? There's already a lot of flags that are used for both.

Reply:
They have totally different meanings and uses and that's ok?

Reply:
Was thinking about "url". Agree we can use bootstrap for both cloud and onprem